src/plugins/mer/mervirtualboxmanager.cpp

```diff
@@ -601,15 +650,18 @@ VirtualMachineInfo virtualMachineInfoFromOutput(const QString &output)
     while ((pos = rexp.indexIn(output, pos)) != -1) {
         pos += rexp.matchedLength();
         if (rexp.cap(0).startsWith(QLatin1String("Forwarding"))) {
+            QString ruleName = rexp.cap(1);
```

Comment:
I think we should either not introduce `ruleName` or replace all occurences of  `rexp.cap(1)` with `ruleName`